Formula/sapling.rb

```diff
@@ -31,19 +31,23 @@ class Sapling < Formula
   depends_on "openssl@1.1"
   depends_on "python@3.11"
 
+  def simplified_version
+    version.to_s.split("-").take(2).join("-")
```

Comment:
```suggestion
    segments = version.to_s.split("-")
    "#{segments.take(2).join("-")}+#{segments.last}"
```
Maybe?

Reply:
Looks fine to me. With this change, `simplified_version` may not be "simplified", though.